DependencyInjection/CmfSeoExtension.php

```diff
@@ -59,9 +59,9 @@ public function load(array $configs, ContainerBuilder $container)
 
         $sonataBundles = array();
         if ($this->isConfigEnabled($container, $config['persistence']['phpcr'])) {
-            $container->setParameter($this->getAlias().'.backend_type_phpcr', true);
+            $container->setParameter('cmf_seo.backend_type_phpcr', true);
```

Comment:
See my first PR comment:

 > Oh, and I made the extension class to not use the `getAlias` method, but just `cmf_seo`. There is 0 change that we're going to change the alias and if we do, we need to do lots of find & replace stuff already.